test/support/data_case.exs

```diff
@@ -15,22 +15,23 @@ defmodule Cassandrax.DataCase do
       end
 
       defp start_test_connection do
-        child = Cassandrax.Supervisor.child_spec(Cassandrax.TestConn, nodes: ["127.0.0.1:9043"])
+        child = Cassandrax.Supervisor.child_spec(Cassandrax.TestConn,
+          nodes: ["127.0.0.1:9042"], username: "cassandra", password: "cassandra")
```

Comment:
I believe we're duplicating this info here and in the configs. Can you try setting it up in one single place?

Reply:
Also, I was using port 9043 because that's the port I setup in `docker-compose.yml`, can we let's stick to that?
The reason is I didn't want to mix the lib testing database with actual projects database

Reply:
I forgot to clean this up while rushing to finish this tonight! I'll remove these. Thanks for pointing out